### DISCUSSION TITLE
Is it possible to use the vite dev server when the RAILS_ENV is not set to development?

### DISCUSSION THREAD

**Discussion by jasonkoon** [ElMassimo/vite_ruby#394] [Q&A] [answered]
How can I use the vite dev server when the RAILS_ENV is not set to development?
For _reasons_ i have RAILS_ENV set to INTEGRATION but want to use the vite dev server (when developing integration tests).

Thanks

**Comment by ElMassimo** [comment, marked as answer]
Hi Jason!

That's [not supported](https://github.com/ElMassimo/vite_ruby/blob/main/vite_ruby/lib/vite_ruby.rb#L110), though you _could_ try setting `VITE_RUBY_MODE` to `test` or `development` in order to force it.

**Reply by jasonkoon** [reply]
Wow, thanks for quick answer!
